docs/pages/custom-data.rst

```diff
@@ -80,6 +80,30 @@ Python's `inspect.getargspec` is used to only pass the arguments declared by the
 function. This means it's not necessary to add a catch all (``**``) keyword
 argument.
 
+The render function can also be used to display the data from two columns in one column. In the following example we can see the field `last_name` is appended to the `name` field using the render function. Note that `value` is return the value in the column and `record` is used to access the values in the `last_name` column::
```

Comment:
We might want to add a note here that _sorting_ is still done on the value.

Reply:
I agree just forgot to do that.